posix_case_conversion/ext/posix_case_conversion/posix_case_conversion.c

```diff
@@ -1,4 +1,3 @@
-#include <wctype.h>
```

Comment:
How does it compile without this line? Did you check on a Linux system?

Reply:
Ah removal of this line was unintentional. Although, it compiles with warning.